ts/screens/PreferencesScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import { Container, Content, H1, Text, View } from "native-base";
+import * as React from "react";
+import PreferenceList from "../components/PreferencesList";
+import ScreenHeader from "../components/ScreenHeader";
+import I18n from "../i18n";
+import { PreferenceItem } from "../types/PreferenceItem";
+
+const preferences: ReadonlyArray<PreferenceItem> = [
+  {
+    id: "email",
+    icon: require("../../img/wallet/icon-avviso-pagopa.png"),
+    valuePreview: "mario.rossi@postaelettronica.it"
+  },
+  {
+    id: "servicesNotifications",
+    icon: require("../../img/wallet/icon-avviso-pagopa.png"),
+    valuePreview: "Inps, Comune di Venezia, ..."
+  },
+  {
+    id: "language",
+    icon: require("../../img/wallet/icon-avviso-pagopa.png"),
+    valuePreview: "Italiano"
+  },
+  {
+    id: "digitalDomicile",
+    icon: require("../../img/wallet/icon-avviso-pagopa.png"),
+    valuePreview: "Nessuna preferenza impostata"
+  }
+];
+
+const PreferencesScreen: React.SFC = () => (
+  <Container>
+    <Content noPadded={true}>
```

Comment:
I think you can remove noPadded={true} here and content={true} in the following View